infra/helper.py

```diff
@@ -193,8 +193,9 @@ def parse_args(parser, args=None):
 
   # Use hacky method for extracting attributes so that ShellTest works.
   # TODO(metzman): Fix this.
-  is_external = getattr(parsed_args, 'is_external', False)
-  build_integration_path = getattr(parsed_args, 'build_integration_path', False)
+  is_external = getattr(parsed_args, 'external', False)
```

Comment:
is the string 'is_external' or 'external' ? in the test, i see is_external=True

Reply:
is_external is the name of the variable in code, 'external' is the name of the argument. This version is correct. Previous one was wrong.
Should I just use `external` everywhere to avoid confusion?